src/persist-cli/src/main.rs

```diff
@@ -134,30 +135,20 @@ fn main() {
         ))
         .expect("failed to init tracing");
 
-    let root_span = info_span!("persistcli");
```

Comment:
ooc: why was this root span removed here, along with the `instrument` calls?

Reply:
They're very long-lived spans, which is apparently an antipattern (in opentel or something). Concretely, the traces I get in local Tempo are complete garbage when this is in there and decent when it's gone. /shrug